src/Adapter/MSTest.TestAdapter/ObjectModel/FixtureTestResult.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.ObjectModel;
+
+[Serializable]
+internal class FixtureTestResult
```

Comment:
I think it would be valuable to also have:
- duration
- stack trace

we also probably need to work on the captured messages.